rpcs3/rpcs3qt/main_window.cpp

```diff
@@ -2035,14 +2081,23 @@ void main_window::dropEvent(QDropEvent* event)
 		m_gameListFrame->Refresh(true);
 		break;
 	case drop_type::drop_game: // import valid games to gamelist (games.yaml)
-		if (Emu.BootGame(sstr(dropPaths.first()), "", true))
 		{
-			gui_log.success("Elf Boot from drag and drop done: %s", sstr(dropPaths.first()));
+			auto result = Emu.BootGame(sstr(dropPaths.first()), "", true);
+			if (result == game_boot_result::no_errors)
```

Comment:
switch this around and you can do
```suggestion
			if (const auto result = ... ; != game_boot_result::no_errors)
```